AGENTS.md

```
@@ -0,0 +1,89 @@
+# AGENTS.md
+
+## What is this repository
+
+This is the **plugin index** for [crane](https://github.com/konveyor/crane) — a Kubernetes migration tool. It contains only YAML manifest files that register crane transform plugins and point to their binary download URLs. There is no plugin source code here.
+
+Crane uses this index to discover and install plugins via `crane plugin-manager`.
+
+## Repository structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)